code/modules/admin/force_event.dm

```diff
@@ -0,0 +1,96 @@
+///Allows an admin to force an event
+/client/proc/forceEvent()
+	set name = "Trigger Event"
+	set category = "Admin.Events"
+
+	if(!holder || !check_rights(R_FUN))
+		return
+
+	holder.forceEvent()
+
+///Opens up the Force Event Panel
+/datum/admins/proc/forceEvent()
+	if(!check_rights(R_FUN))
+		return
+
+	var/datum/force_event/ui = new(usr)
+	ui.ui_interact(usr)
+
+/// Force Event Panel
+/datum/force_event
+
+/datum/force_event/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "ForceEvent")
+		ui.open()
+
+/datum/force_event/ui_state(mob/user)
+	return GLOB.fun_state
+
+/datum/force_event/ui_static_data(mob/user)
+	var/static/list/category_to_icons
+	if(!category_to_icons)
+		category_to_icons = list(
+			EVENT_CATEGORY_AI = "robot",
+			EVENT_CATEGORY_ANOMALIES = "cloud-bolt",
+			EVENT_CATEGORY_BUREAUCRATIC = "print",
+			EVENT_CATEGORY_ENGINEERING = "wrench",
+			EVENT_CATEGORY_ENTITIES = "ghost",
+			EVENT_CATEGORY_FRIENDLY = "face-smile",
```

Comment:
from what I saw in the testing evidence, this is not an icon that exists, since it wasn't showing up.

Reply:
update fontawesome so they appear in 866157b

Reply:
Does this break old icons? I know fontawesome likes renaming stuff bwtween versions so a little concerned